content/21.nutraceuticals.md

```diff
@@ -210,10 +211,11 @@ Indeed, studies are beginning to investigate whether there is any prophylactic a
 A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than those of SARS-CoV-2 negative patients (24.6 ng/ml; _p_ = 0.004), an association that held when stratifying patients greater than 70 years old [@doi:10.3390/nu12051359].
 These findings seem to be supported by a Belgian observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia, where 25-hydroxyvitamin D plasma concentrations were measured and CT scans of the lungs were obtained upon hospitalization [@doi:10.1101/2020.05.01.20079376].
 A significant difference in 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched diseased controls.
```

Comment:
```suggestion
A significant difference in 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched hospitalized controls.
```
I think this might be more clear than the phrase they use? Just an idea to swap it.

Reply:
The weird thing from the paper is that they never specify who these people are or what disease they have... at any stage. We can only assume they mean hospitalized